src/server/game/Entities/Unit/Unit.h

```diff
@@ -2030,10 +2030,12 @@ class TC_GAME_API Unit : public WorldObject
         void _RegisterDynObject(DynamicObject* dynObj);
         void _UnregisterDynObject(DynamicObject* dynObj);
         DynamicObject* GetDynObject(uint32 spellId);
+        std::list<DynamicObject*> GetDynObjects(uint32 spellId);
```

Comment:
is there any reason for using std::list over std::vector? if not then use vector